docs/tools.md

```
@@ -0,0 +1,1113 @@
+```
+
+When this tool is registered, FastMCP automatically:
+
+* Uses the function name (`add`) as the tool name.
+* Uses the function's docstring (`Adds two integer numbers...`) as the tool description.
+* Generates an input schema based on the function's parameters and type annotations.
+* Handles parameter validation and error reporting.
+
+The way you define your Python function dictates how the tool appears and behaves for the LLM client.
+
+<Tip>
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: Tip

(MD033, no-inline-html)